src/components/SidePanels/TabPanels/utils.ts

```diff
@@ -0,0 +1,461 @@
+import { FC, MutableRefObject, ReactNode } from 'react';
+import { clamp } from '../../../utils/utilities';
+import { DEFAULT_PANEL_HEIGHT, DEFAULT_PANEL_MAX_HEIGHT, DEFAULT_PANEL_MAX_WIDTH, DEFAULT_PANEL_MIN_HEIGHT, DEFAULT_PANEL_WIDTH, PANEL_HEADER_HEIGHT } from '../constants';
+import { Comments, History, Info, Relations } from '../DetailsPanel/DetailsPanel';
+import { OutlinerComponent } from '../OutlinerPanel/OutlinerPanel';
+import { PanelProps } from '../PanelBase';
+import { emptyPanel, JoinOrder, PanelBBox, PanelView, Side, ViewportSize } from './types';
+
+export const lastCallTime: { [key: string]: number } | undefined = {};
```

Comment:
This cannot be undefined as it is initialized as const as an empty object. Were you wanting the type here to be optional key? 

Maybe something more like this:

```suggestion
export const lastCallTime: Record<string, number | undefined> = {};
```

Reply:
actually this was a leftover and is not used, deleted